app/controllers/consolidations/DisassociateUCRSummaryController.scala

```diff
@@ -32,12 +32,12 @@ import scala.concurrent.ExecutionContext
 
 @Singleton
 class DisassociateUCRSummaryController @Inject()(
-  authenticate: AuthenticatedAction,
-  getJourney: JourneyRefiner,
-  mcc: MessagesControllerComponents,
-  submissionService: SubmissionService,
-  movementRepository: MovementRepository,
-  page: disassociate_ucr_summary
+                                                  authenticate: AuthenticatedAction,
+                                                  getJourney: JourneyRefiner,
+                                                  mcc: MessagesControllerComponents,
+                                                  submissionService: SubmissionService,
+                                                  movementRepository: CacheRepository,
```

Comment:
What about name here? should it be `cache`?

Reply:
Yeh think I've missed a few of these